python/cudf/cudf/tests/test_parquet.py

```diff
@@ -125,6 +128,11 @@ def parquet_file(request, tmp_path_factory, pdf):
     return fname
 
 
+@pytest.fixture(scope="module")
+def seed():
+    return 42
```

Comment:
We may want an environment variable to override this to allow for testing more than seed `42` without having to modify source code 😆